mllib/src/main/scala/org/apache/spark/ml/image/HadoopUtils.scala

```diff
@@ -42,9 +42,11 @@ private object RecursiveFlag {
     val old = Option(hadoopConf.get(flagName))
     hadoopConf.set(flagName, value.toString)
     try f finally {
-      old match {
-        case Some(v) => hadoopConf.set(flagName, v)
-        case None => hadoopConf.unset(flagName)
+      // avoid false positive of DLS_DEAD_LOCAL_STORE_IN_RETURN by SpotBugs
```

Comment:
is it a real problem?

Reply:
It is not a real problem. However, I cannot find to disable only this check in `findPuzzles`.